src/dmd/dimport.d

```diff
@@ -160,15 +160,22 @@ extern (C++) final class Import : Dsymbol
                 {
                     if (p.isPkgMod == PKG.unknown)
                     {
+                        uint preverrors = global.errors;
                         mod = Module.load(loc, packages, id);
                         if (!mod)
                             p.isPkgMod = PKG.package_;
                         else
                         {
                             // mod is a package.d, or a normal module which conflicts with the package name.
```

Comment:
Does this comment still apply?

Reply:
I think yes, the first part is the true and the following is the error.